### DISCUSSION TITLE
Integration into IDE

### DISCUSSION THREAD

**Discussion by vandres** [spatie/ray#894] [Ideas]
I would wish for a IDE integration of Ray. I am using PHPStorm and it would be great, if the logging would happen in a IDE window. Something like the terminal in the screenshot here:

![grafik](https://github.com/spatie/ray/assets/1436383/070b4d50-8e0a-40b7-8c29-54ca4e0e1736)

At the moment the Ray window is somehow always in the way of another window.

A browser plugin would also be great ;)

**Comment by woodymendoza** [comment]
Same here. An IDE integration would be great, but ultimately it would be even better if PHPStorm had a way to embed/snap external apps in a panel.  

cc: @freekmurze @brendt

**Comment by woodymendoza** [comment]
Maybe a TUI for RAY would be easier.
